libraries/ESP8266WebServer/examples/AdvancedWebServer/AdvancedWebServer.ino

```diff
@@ -94,6 +94,24 @@ void handleNotFound() {
   digitalWrite(led, 0);
 }
 
+void drawGraph() {
+  String out = "";
```

Comment:
no empty string assign

Reply:
also, this series of concats is very bad practice. Let's add a call to reserve(), i think 2550 should be enough to contain everything (approximate, I eyeballed it).